corehq/apps/hqwebapp/templatetags/hq_shared_tags.py

```diff
@@ -625,6 +625,15 @@ def render(self, context):
     return AddToBlockNode(nodelist, 'js-inline')
 
 
+@register.simple_tag
+def html_attr(value):
+    if isinstance(value, basestring):
+        value = json.dumps(value)[1:-1]
```

Comment:
How is this consumed in JS? Are the quotes re-added and HTML entities un-escaped before JSON parsing? I'm thinking of a scenario like this:
```python
>>> import json
>>> from django.utils.html import escape
>>> escape(json.dumps('abc"def')[1:-1])
u'abc\\&quot;def'
```
Not sure what string value that will produce in JS. Will the bashslash be removed? Maybe it would be better to `escape()` the string before `json.dumps()`?

Relevant: https://stackoverflow.com/a/4015380/10840